documentation/docs/steps/neoDeploy.md

```diff
@@ -54,3 +54,50 @@ steps:
       account: <myDeployAccount>
       host: hana.example.org
 ```
+
+## Example for invalidating the cache
+
+* Setting parameter `invalidateCache` to `true`, will clean up the cache of a Fiori Launchpad site, refreshing the content of any applications deployed there. This is **applicable** only when the application is deployed and accessed via Portal service (Fiori Launchpad site).
+
+Setting this parameter to true, needs additional configurations to go through. Firstly, create an oauth credential as shown below:
+
+### OAuth credential creation
+
+  1 Select the oauth settings from your subaccount, create a new client with subscription to portal/nwc as shown in the following images:
+
+    ![OAuth client creation](../images/oauthClientCreation.png)
+
+  2 Select the "Clients" tab, which provides an option to register a new client. Click on "Register New Client" button.
+
+  3 After clicking, in the subscription field, select the portal landscape you would like to subscribe to, ex: `portal/nwc` or `portal/sandbox` as shown below:
+
+    ![Portal subscription](../images/portalSubscription.png)
+
+  4 In the "Authorization Grant" field, select "Client Credentials" from the drop down menu. Then, enter the "Secret" field with a user defined password and finally save the changes.
+
+After saving these changes, create a UsernamePassword type credential with client Id (obtained from registering new client) as username and client secret (Password entered in the secret field above) as password in Jenkins.
+
+### Set Site Id
+
+After login to the portal service from SAP CP, the user has a possibility to create a site directory, within which the deployed application resides.
+
+Click on "+" tile, to create a new site directory. After saving the changes to a new site directory tile, the control is navigated to a new page which holds the "site id" in the url. By default, site is set as default as shown below image:
+
+![Site set to default](../images/portalSiteSetToDefault.png)
+
+If it is already set to default, no need to configure `siteId` in the configuration file. If not set to default, please configure `siteId`, as shown in the below configuration:
+
+```groovy
+neoDeploy script: this, neo: [credentialsId: 'my-credentials-id', host: hana.example.org, account: 'testAccount']
```

Comment:
How about only having the step configured with parameters and not show a config.yml example. I find it confusing with some of the configuration in the step and other configuration in the yaml. How about showing the step configured only with parameters.

Else please add the words: "Example configuration:" as is done in the above example.

Reply:
@o-liver Is it ok if I improve the Docu in another PR? This is getting so huge that I cannot follow on your comments. If the request is on docu, I would like to handle it in another PR.